db/src/Pos/DB/Rocks/Functions.hs

```diff
@@ -101,6 +108,9 @@ openNodeDBs recreate fp = do
     ensureDirectoryExists :: MonadIO m => FilePath -> m ()
     ensureDirectoryExists = liftIO . createDirectoryIfMissing True
 
+    ensureEmptyFileExists :: MonadIO m => FilePath -> m ()
+    ensureEmptyFileExists file = liftIO $ withFile file AppendMode (\_ -> return ())
```

Comment:
This is already on `develop`!
https://github.com/input-output-hk/cardano-sl/blob/develop/db/src/Pos/DB/Rocks/Functions.hs#L111-L112